store/copr/batch_coprocessor.go

```diff
@@ -542,6 +566,52 @@ func buildBatchCopTasksForPartitionedTable(bo *backoff.Backoffer, store *kvStore
 	return batchTasks, nil
 }
 
+func buildBatchCopTasksConsistentHash(bo *backoff.Backoffer, store *kvStore, rangesForEachPhysicalTable []*KeyRanges, storeType kv.StoreType, mppStoreLastFailTime *sync.Map, ttl time.Duration, balanceWithContinuity bool, balanceContinuousRegionCount int64) ([]*batchCopTask, error) {
+	batchTasks, err := buildBatchCopTasksCore(bo, store, rangesForEachPhysicalTable, storeType, mppStoreLastFailTime, ttl, balanceWithContinuity, balanceContinuousRegionCount)
+	if err != nil {
+		return nil, err
+	}
+	cache := store.GetRegionCache()
+	stores, err := cache.GetTiFlashComputeStores(bo.TiKVBackoffer())
+	if err != nil {
+		return nil, err
+	}
+	if len(stores) == 0 {
+		return nil, errors.New("Number of tiflash_compute node is zero")
```

Comment:
Will this be better?
```suggestion
		return nil, errors.New("No available tiflash_compute node")
```

Reply:
Done